lib/generators/arclight/templates/catalog_controller.rb

```diff
@@ -137,9 +137,16 @@ class CatalogController < ApplicationController
     ##
     # Configuration for partials
     config.index.partials.insert(0, :index_breadcrumb)
-    
-    config.show.metadata_partials = [:summary_field, :access_field]
-    
+
+    config.show.metadata_partials = [
+      :summary_field,
+      :access_field,
+      :background_field,
+      :scope_and_arrangement_field,
+      :related_field,
+      :admin_info_field
+    ]
```

Comment:
I'm asking if changing 
```ruby
config.show.metadata_partials = [...]
```
to
```ruby
config.show.metadata_partials = Arclight::Engine.config.catalog_controller_field_accessors
```
would make sense so you don't have to add the same symbol to both arrays in both files as part of the process of adding a new section (this might just be particularly obvious to me since we just had to do that multiple times for this PR).

That's why I asked the question re: the likelihood of these two arrays diverging in the near future (which it's sounding like you believe they will).